pkg/security/probe/kfilters/policy.go

```diff
@@ -59,15 +59,26 @@ func (m PolicyMode) MarshalJSON() ([]byte, error) {
 
 // MarshalJSON returns the JSON encoding of the policy flags
 func (f PolicyFlag) MarshalJSON() ([]byte, error) {
+	flags := f.StringArray()
+
+	for idx, flag := range flags {
+		flags[idx] = "\"" + flag + "\""
+	}
+
+	return []byte("[" + strings.Join(flags, ",") + "]"), nil
```

Comment:
why not using `json.Marshal` directly ? is this performance sensitive ?
https://go.dev/play/p/BxQAehD1wj5

Reply:
good catch!